Omulosi/iReporter
# #162660021 Update createdon and createdby fields of a record

#### What does this PR do?
#### Description of Task to be completed?
* update the createdon fields of all records obtained from the database to contain a string representation of the date. 
* create a new records table with a created by field and update database models to include the new field in the query to store data to the database.
#### How should this be manually tested?
clone the repository and checkout the bg-update-record-fields-162660021. Run the application as documented in the README. Create a user and add an intervention record. Fetch the record created. The response should contain a string formatted createdon field and a createdby field containing id of the creator.
#### What are the relevant pivotal tracker stories?
#162660021
